### ISSUE TITLE
Q: Installation script returns curl: no URL specified!

### ISSUE THREAD

**Issue by Feelixe-tin** [docker/compose-cli#1737]
I have the following job set up on CircleCI. It worked yesterday but today I get the error below. Since it's a completely clean ubuntu instance I'm assuming there's something wrong with the installation script? 
```
machine:
  image: ubuntu-2004:202010-01
steps:
  - checkout
  - run:
    name: "Install Docker Compose CLI"
    command: |
      curl -L https://raw.githubusercontent.com/docker/compose-cli/v1.0.16/scripts/install/install_linux.sh | sh
```

```
#!/bin/bash -eo pipefail
curl -L https://raw.githubusercontent.com/docker/compose-cli/v1.0.16/scripts/install/install_linux.sh | sh

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  6325  100  6325   Running checks...    0 --:--:-- --:--:-- --:--:--     0
100  6325  100  6325    0     0  39043      0 --:--:-- --:--:-- --:--:-- 38803
Checks passed!
Downloading CLI...
curl: no URL specified!
curl: try 'curl --help' or 'curl --manual' for more information

Exited with code exit status 2
CircleCI received exit code 2
```

**Comment by ndeloof** [comment]
Hi,

we indeed released 1.0.17 yesterday with updated packaging, and this unfortunately make it incompatible with previous install script if used without and explicit download URL. As script was designed to get "latest release" there's no safe way to change packaging :'(

As an alternative you can either
- run install_linux.sh from v1.0.16 tag with explicit `DOWNLOAD_URL=https://github.com/docker/compose-cli/releases/download/v1.0.16/docker-linux-amd64.tar.gz`
- switch back to install script from `main` branch, which will install compose-cli "Cloud Integrations" 1.0.17, but won't install Compose V2.

If what you'd like is to get Compose V2 better download the release binary:
`curl -L https://github.com/docker/compose-cli/releases/download/v2.0.0-beta.3/docker-compose-linux-amd64 -o ~/.docker/cli-plugins/docker-compose`

**Comment by Feelixe-tin** [comment]
That works, thank you❤
